routing/routing_integration_tests/road_graph_tests.cpp

```diff
@@ -0,0 +1,47 @@
+#include "testing/testing.hpp"
+
+#include "platform/local_country_file.hpp"
+
+#include "geometry/point2d.hpp"
+
+#include "indexer/classificator_loader.hpp"
+#include "indexer/feature_altitude.hpp"
+#include "indexer/index.hpp"
+#include "indexer/mwm_set.hpp"
+
+#include "routing_common/car_model.hpp"
+#include "routing/features_road_graph.hpp"
+#include "routing/road_graph.hpp"
+
+#include "routing_integration_tests/routing_test_tools.hpp"
+
+#include <vector>
+
+using namespace routing;
+using namespace integration;
+
+// The test on combinatorial explosion of number of fake edges at FeaturesRoadGraph.
+// It might happen when a lot of roads are intersected at one point. For example,
```

Comment:
s/are intersected/intersect/

Reply:
Is "intersect" an adjective? As far as I'm aware it's a verb: http://www.dictionary.com/browse/intersect

Reply:
"roads are intersected" requires an object, i.e. you must specify what they are intersected by
cf. https://www.google.com/search?&q="several+lines+intersect"&oq="several+lines+intersect"

Reply:
Yes. I see. Fixed.